affinity/cpp-23/dXXXXr0.md

```diff
@@ -0,0 +1,295 @@
+# DXXXXr0: Policy property for describing adjacency
+
+**Date: 2020-04-06**
+
+**Audience: SG1, SG14**
+
+**Authors: Gordon Brown, Ruyman Reyes, Michael Wong, H. Carter Edwards, Thomas
+Rodgers, Mark Hoemmen, Tom Scogland**
+
+**Emails: gordon@codeplay.com, ruyman@codeplay.com, michael@codeplay.com,
+hedwards@nvidia.com, rodgert@twrodgers.com, mhoemme@sandia.gov,
```

Comment:
Please change my e-mail address to `mhoemmen@stellarscience.com` -- thanks!

Reply:
Done.